packages/expo-media-library/android/src/test/java/expo/modules/medialibrary/assets/GetAssetsTest.kt

```diff
@@ -62,7 +57,7 @@ internal class GetAssetsTest {
   }
 
   @Test
-  fun `getAssets should resolve with correct response`() {
+  fun `getAssets should resolve with correct response`() = runBlocking {
```

Comment:
```suggestion
  fun `getAssets should resolve with correct response`() = runTest {
```
Let's use `runTest` from the coroutines test library instead of `runBlocking`. We can add this to all the tests that need it